lib/plugins/aws/provider.js

```diff
@@ -480,6 +480,18 @@ class AwsProvider {
             required: ['Fn::Sub'],
             additionalProperties: false,
           },
+          awsCfBase64: {
+            type: 'object',
+            properties: {
+              'Fn::Base64': {
+                anyOf: [
+                  { type: 'string' },
+                  { $ref: '#/definitions/awsCfFunction' },
+                  { $ref: '#/definitions/awsCfToJsonString' },
+                ],
```

Comment:
Maybe we can simplify it to:

```
'Fn::Base64': { $ref: '#/definitions/awsCfInstruction' }
```

And include `awsCfBase64` in `awsCfFunction` definition ?

Reply:
I guess you meant to include `awsCfToJsonString` in `awsCfFunction`? Although `awsCfBase64` itself could be included as well perhaps? Recursive, but perhaps valid?

Reply:
Yes, it'll be good as well.. I think this config got out of hand a bit. 

We cannot validate all value constraints as it's done on AWS side, so I believe here best is just allow any kind of CF functions in values that go as-is to CF template. And easiest way to achieve that is probably just group them all at `awsCfFunction`

Reply:
Updated ✅

Reply:
@medikoo Please let me know if you agree with this approach. It's no silver bullet, but still an improvement. Does anything remain to be done?